token/cli/src/main.rs

```diff
@@ -100,7 +101,39 @@ pub const MULTISIG_SIGNER_ARG: ArgConstant<'static> = ArgConstant {
     help: "Member signer of a multisig account",
 };
 
-pub const CREATE_TOKEN: &str = "create-token";
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Display, EnumString, IntoStaticStr)]
+#[strum(serialize_all = "kebab-case")]
+pub enum CommandName {
+    CreateToken,
+    Close,
+    Bench,
+    CreateAccount,
+    CreateMultisig,
+    Authorize,
+    Transfer,
+    Burn,
+    Mint,
+    Freeze,
+    Thaw,
+    Wrap,
+    Unwrap,
+    Approve,
+    Revoke,
+    Balance,
+    Supply,
+    Accounts,
+    Address,
+    AccountInfo,
+    MultisigInfo,
+    Gc,
+    SyncNative
```

Comment:
Eventually, the idea for these is to be completely expressive with the values required for the command encoded as well.

For example, check out the CLI commands in the main repo: https://github.com/solana-labs/solana/blob/c419845cfeb1c7ab9ca959de66e90b9dacbc25b7/cli/src/cli.rs#L47

But we can definitely start here :smile: